app/assets/stylesheets/authors.css.scss

```diff
@@ -0,0 +1,13 @@
+.author {
+  clear: both;
+  .author_books {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector `author_books` should be written in lowercase with hyphens